get-users/cross-promotion.md

```diff
@@ -0,0 +1,23 @@
+# Cross-Promotion
+
+If you have more than one application you can use Tap for Tap to cross promote within your own network. Do this by subdividing your Get User traffic between showing ads for other developers and self promoting your own applications.
+
+In order to cross promote you will need to:
+
+1. Download our [SDK](/doc/getting-started)
+2. Set your Get Users traffic to Self Promotion > 0% 
+3. Upload promotional [creatives](doc/get-users/creatives-guidelines) for your applications
+
+
+## Self vs. External Preference
+
+Once you’ve chosen how much traffic you want to go to the Tap Exchange, you can further divide that traffic to promote other developers or to promote yourself.
+
+
+## Earning Credits
+
+When using the tap exchange to cross promote your own network, you do not earn and spend credits. Cross promotion is simply a free way to leverage your ad inventory to promote yourself. If you have an app that you would like to promote more than others, you can control this by adjusting the spend priority under your accounts page. 
```

Comment:
Sometimes Tap Exchange is capitalized, sometimes not. Pick a standard? I think capitalized, since it's a name of a system?